app/src/test/java/com/ikarabulut/GameDisplayTest.java

```diff
@@ -1,41 +1,112 @@
 package com.ikarabulut;
 
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class GameDisplayTest {
+
+    private final PrintStream standardOut = System.out;
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+    private static GameDisplay gameDisplay;
+
+    @BeforeAll
+    static void setByteArrayOutputStream() {
+        gameDisplay = new GameDisplay();
+    }
+
+    @BeforeEach
+    void setUp() {
+        System.setOut(new PrintStream(outputStreamCaptor));
+    }
+
+    @AfterEach
```

Comment:
@jramnani does this solve this issue [https://github.com/ikarabulut/JavaGameOfLife/pull/8#discussion_r877373384]